src/test/java/com/google/devtools/build/lib/pkgcache/BUILD

```diff
@@ -278,10 +278,10 @@ java_library(
     ],
 )
 
-java_test(
-    name = "LoadingPhaseRunnerTest",
-    test_class = "com.google.devtools.build.lib.pkgcache.LoadingPhaseRunnerTest",
-    runtime_deps = [
-        ":LoadingPhaseRunnerTest_lib",
-    ],
-)
+# java_test(
```

Comment:
Should this be removed, or is the test going to be fixed?
Prefer tagging as "noci" if the test doesn't pass on CI.